auslib/db.py

```diff
@@ -637,6 +637,7 @@ class Rules(AUSTable):
     def __init__(self, metadata, dialect):
         self.table = Table('rules', metadata,
                            Column('rule_id', Integer, primary_key=True, autoincrement=True),
+                           Column('alias', String(50), unique=True),
```

Comment:
Doesn't look like one was created when I upgraded my local db. I'll update the migrate script to add it.